maps/away/bearcat/bearcat.dm

```diff
@@ -16,7 +16,7 @@
 	)
 
 /obj/effect/overmap/ship/bearcat
-	name = "light freighter"
+	classification = "Light Freighter"
```

Comment:
Capitalizing these makes them proper nouns, getting you "Bearcat, Light Freighter". Depending on what you want,
```suggestion
	classification = "light freighter"
```
or
```suggestion
	classification = "\improper Light Freighter"
```

Reply:
Went with the first option and added the upper case helper to the nanoUI to compensate in 9b46ce7fe1b09c41eea235645a4927e7ecc690c9 
Thanks!